delete.php

```diff
@@ -1,13 +1,13 @@
 <?php
 
-
 if (!isset($_POST['post'])) {
   die('Javascript needs to be enabled for you to delete keys.');
 }
 
-
 require_once 'includes/common.inc.php';
 
+global $redis;
+global $server;
```

Comment:
I don't understand why all these `global` statements are needed? These files aren't included within functions?

Reply:
PHPStorm syntax analyzer recommend use global statement for explicit declaring that variable is global (and declared before)